data/src/main/java/com/mobgen/data/SharedPreferencesManager.kt

```diff
@@ -0,0 +1,6 @@
+package com.mobgen.data
+
+interface SharedPreferencesManager {
```

Comment:
The correct name for this is something related to Auth instead of SharedPreferences.